exercises/Descartes_Planning_and_Execution/solution_ws/src/ur5_demo_descartes/include/ur5_demo_descartes/ur_moveit_plugin.h

```diff
@@ -94,8 +94,8 @@
 #include <kdl/chainiksolvervel_pinv.hpp>
 #include <kdl/chainiksolverpos_nr_jl.hpp>
 #include <kdl/chainfksolverpos_recursive.hpp>
-#include <moveit/kdl_kinematics_plugin/chainiksolver_pos_nr_jl_mimic.hpp>
-#include <moveit/kdl_kinematics_plugin/chainiksolver_vel_pinv_mimic.hpp>
+//#include <moveit/kdl_kinematics_plugin/chainiksolver_pos_nr_jl_mimic.hpp>
+//#include <moveit/kdl_kinematics_plugin/chainiksolver_vel_pinv_mimic.hpp>
```

Comment:
can you delete these lines instead?

Reply:
fixed in commit f15a33d.  Please re-review.